src/core/core.scale.js

```diff
@@ -100,6 +102,23 @@ function computeTextSize(context, tick, font) {
 }
 
 module.exports = Element.extend({
+	/**
+	 * @private
+	 */
+	_configure: function() {
+		var me = this;
+		var opts = me.options;
+		var ttOpts = opts.tooltips || {};
+		var tickOpts = opts.ticks;
+		var global = defaults.global;
```

Comment:
you could move this line up to the top of the file and then share with `draw` where it's also being used